cypress/page/products-list.page.ts

```diff
@@ -3,9 +3,8 @@ class ProductsListPage {
   private proceedCheckoutBtn: string;
 
   constructor() {
-    this.addCartBtn =
-      '#center_column a.button.ajax_add_to_cart_button.btn.btn-default';
-    this.proceedCheckoutBtn = '#layer_cart a';
+    this.addCartBtn = '.button-container .ajax_add_to_cart_button';
+    this.proceedCheckoutBtn = '.button-container a.button-medium';
```

Comment:
here probably you can use a contains with the word "Proceed to checkout" or use an attribute of a tag to indicate we are looking for the Proceed to checkout button